src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const tempArr = date.split(fromFormat[3]);
+  const resultArr = [];
+
+  for (let i = 0; i <= 2; i++) {
+    for (let n = 0; n <= 2; n++) {
+      if (toFormat[i][0] === fromFormat[n][0]) {
+        if (toFormat[i].length < fromFormat[n].length) {
+          resultArr[i] = +tempArr[n] % 100;
```

Comment:
The conversion logic for two-digit years assumes that any two-digit year is in the 20th century if it's less than 30. This assumption might not be valid for all use cases. Consider making this logic more flexible or configurable.